CMakeLists.txt

```diff
@@ -11,30 +11,40 @@ project(soem_robotiq_drivers)
 
  
 # Do setup in case of ros package, If ROS_ROOT is set, it is                                                                                                 # recommended to use RTT/OCL through the ros packages.                                                                                                       
-set (ROS_ROOT $ENV{ROS_ROOT} )
-if (ROS_ROOT)
-  include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
-  rosbuild_init()
-  rosbuild_find_ros_package( rtt )
-  set( RTT_HINTS HINTS ${rtt_PACKAGE_PATH}/../install )
-endif()
-
 #uncomment if you have defined messages
-rosbuild_genmsg()
-
+if(NOT ORO_USE_ROSBUILD)
+  find_package(catkin REQUIRED message_generation rtt_roscomm)
+  add_message_files( FILES
+    FingerMsg.msg
+    RobotiqMsg.msg
+    )
+
+  generate_messages()
+endif()
 # Set the CMAKE_PREFIX_PATH in case you're not using Orocos through
 # ROS for helping these find commands find RTT.
 find_package(OROCOS-RTT REQUIRED ${RTT_HINTS} )
-
 # Defines the orocos_* cmake macros. See that file for additional
 # documentation.
 include(${OROCOS-RTT_USE_FILE_PATH}/UseOROCOS-RTT.cmake)
```

Comment:
I think it's better to move the find_package(OROCOS-RTT) and include(.../UseOrocos.cmake) above line 15 because of the UseOrocos might set ORO_USE_ROSBUILD depending on how this cmake file has been called: https://github.com/orocos-toolchain/rtt/blob/master/UseOROCOS-RTT.cmake#L46-L48